palace/CMakeLists.txt

```diff
@@ -132,7 +132,7 @@ add_subdirectory(models)
 add_subdirectory(utils)
 
 # Add binary extension for build architecture
-if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64|arm")
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|arm")
```

Comment:
Seeing a few of these ${} removals around the cmake files, looks like it's related to cmake's string/list expansion rules, out of interest why the change?

Reply:
Just trying to be consistent across all the code. CMake `if` supports the first argument being a string or a variable so I've just expanded out all the strings for `STREQUAL` versus using the variable for `MATCHES`. In hindsight I think just always using the variable for `if` is better so I'll go make that change.

Reply:
Fixed more completely in [aa70d1e](https://github.com/awslabs/palace/pull/58/commits/aa70d1e7c681917ab5f2e5858aa035a36dd3fb70).